core/app/modules/import-export/module.php

```diff
@@ -373,7 +373,7 @@ private function get_config_data() {
 			'exportURL' => $export_url,
 			'summaryTitles' => $this->get_summary_titles(),
 			'isUnfilteredFilesEnabled' => Uploads_Manager::are_unfiltered_uploads_enabled(),
-			'editElementorHomePageUrl' => $this->get_edit_elementor_home_page_url(),
+			'editElementorHomePageUrl' => $this->get_elementor_home_page_url(),
```

Comment:
Please change  `editElementorHomePageUrl` to a more descrish string, for example `ElementorHomePageUrl`.
Please notice that changes should be made in the following files as well:

-  `core/app/modules/import-export/assets/js/shared/kit-data/kit-data.js` 
-  `core/app/modules/import-export/assets/js/pages/import/import-complete/import-complete.js`